tests/jballerina-unit-test/src/test/resources/test-src/annotations/display_annot.bal

```diff
@@ -14,15 +14,33 @@
 // specific language governing permissions and limitations
 // under the License.
 
-@icon { path: "/fooIconPath.icon" }
+@display {iconPath: "fooIconPath.icon", label: "Foo function"}
```

Comment:
Shall we add a test for the openness of `DisplayData` too? With additional fields in the annotation?

Reply:
Done